security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -644,6 +645,20 @@
                 <dns_nsupdate_key type="TextField">
                     <Required>N</Required>
                 </dns_nsupdate_key>
+                <dns_opnsense_host type="TextField">
+                    <Required>N</Required>
+                    <default>localhost:443</default>
```

Comment:
that's not a host - it is a hostname and a port.

Reply:
I know, but I didn't know how to name it. Iinsert the key and token between "https://" and hostname, that is why I didn't used the full URL.

Reply:
You could separate them to two fields.

Reply:
I'll do it on friday

Reply:
done